README.md

```diff
@@ -313,6 +313,75 @@ You can configure the OpenTelemetry SDK for applications which can't currently b
 instrumentation.opentelemetry.io/inject-sdk: "true"
 ```
 
+### Target Allocator
+
+The OpenTelemetry Operator comes with an optional component, the Target Allocator (TA). When creating an OpenTelemetryCollector Custom Resource (CR) and setting the TA as enabled, the Operator will create a new deployment and service to serve specific `http_sd_config` directives for each Collector pod as part of that CR. It will also change the Prometheus receiver configuration in the CR, so that it uses the `http_sd_config` from the TA. The following example shows how to get started with the Target Allocator:
+
+```yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: collector-with-ta
+spec:
+  mode: statefulset
+  targetAllocator:
+    enabled: true
+  config: |
+    receivers:
+      prometheus:
+        config:
+          scrape_configs:
+          - job_name: 'otel-collector'
+            scrape_interval: 10s
+            static_configs:
+            - targets: [ '0.0.0.0:8888' ]
+
+    exporters:
+      logging:
+
+    service:
+      pipelines:
+        traces:
+          receivers: [prometheus]
+          processors: []
+          exporters: [logging]
+```
+
+Behind the scenes, the OpenTelemetry Operator will convert the Collector’s configuration after the reconciliation into the following:
```

Comment:
The Operator also updates the [Prometheus receiver options](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/72799c31e8156923b7e080861117af84044325cc/receiver/prometheusreceiver#opentelemetry-operator) right?

---
![grafik](https://user-images.githubusercontent.com/10403402/213176883-c7d2e834-9d78-4c20-8d9d-e5e18f0665c6.png)